src/converse-message-view.js

```diff
@@ -260,6 +260,7 @@ converse.plugins.add('converse-message-view', {
                         'occupant': this.model.occupant,
                         'pretty_time': time.format(api.settings.get('time_format')),
                         'retraction_text': is_retracted ? this.getRetractionText() : null,
+                        'first_unread': this.model.get('first_unread'),
```

Comment:
This line isn't necessary. `first_unread` already gets passed in via `this.model.toJSON()` on line 251.

Reply:
ok. I will remove it